client/state/selectors/test/get-post-revision-changes.js

```diff
@@ -59,11 +64,15 @@ describe( 'getPostRevisionChanges', () => {
 	};
 
 	test( 'should return an empty array if the revision is not found', () => {
```

Comment:
Nit: Test description is no longer true of the logic. maybe 'default object' or something similar to denote that the object does have values, but those are empty.